text/0069-api-v2.md

```diff
@@ -0,0 +1,135 @@
+# TiKV API V2
+
+## Motivation
```

Comment:
we need a compatibility section to list the potential compatibility issues and the solution to solve them. For compatibility, the following things need to be considered:

1. compatibility with other components, like BR, TiCDC
2. compatibility with other features, like placement rules and other scheduling constraints and policies.

BTW, it's better to add a testing section to describe how to test these API changes on correctness and performance.

Reply:
done